docs/source/using/host-folders.adoc

```diff
@@ -12,60 +12,71 @@ toc::[]
 
 Host folders are directories on the host which are shared between the host and the Minishift VM.
 They allow for a two way file synchronization between host and VM.
-The following sections discuss the various types of host folders, driver provided host folders, as well as the Minishift `hostfolder` command.
+The following sections discuss the various types of host folders, driver provided host folders, as well as the `minishift hostfolder` command.
```

Comment:
I made similar changes in lots of places where people wrote only the sub-command without the parent, and I did this because of the likelihood that someone will copy and paste the command text, and if we only write something like "get" instead of "minishift config get" then the command wouldn't work.